library/src/helpers/schema.ts

```diff
@@ -206,34 +209,125 @@ export class SchemaHelpers {
     'object',
     'null',
   ];
+  private static jsonSchemaKeywordTypes: Record<string, string> = {
+    // string
+    maxLength: 'string',
+    minLength: 'string',
+    pattern: 'string',
+    contentMediaType: 'string',
+    contentEncoding: 'string',
+    // number
+    multipleOf: 'number',
+    maximum: 'number',
+    exclusiveMaximum: 'number',
+    minimum: 'number',
+    exclusiveMinimum: 'number',
+    // array
+    items: 'array',
+    maxItems: 'array',
+    minItems: 'array',
+    uniqueItems: 'array',
+    contains: 'array',
+    maxContains: 'array',
+    minContains: 'array',
+    prefixItems: 'array',
+    additionalItems: 'array',
+    unevaluatedItems: 'array',
+    // object
+    maxProperties: 'object',
+    minProperties: 'object',
+    required: 'object',
+    properties: 'object',
+    patternProperties: 'object',
+    propertyNames: 'object',
+    dependencies: 'object',
+    dependentRequired: 'object',
+    dependentSchemas: 'object',
```

Comment:
Why does this define keywords not located in JSON Schema?

Reply:
They are, but in 2012-12 draft 😅 Perhaps we should only focus of draft-07, ok I will remove it :)

Reply:
We are only using draft 7 in the spec (at the moment) 🙂

Reply:
Yep, but I like new keywords from schema :D

Reply:
I mean whether or not you like them shouldn't really matter 😄 If you should not be able to define them in the spec why should they be accounted for in the code?

Reply:
I've already prepared for the upgrade to draft 2021 😄

Reply:
Removed :)

Reply:
Ahh didn't see it. 

But yea, be careful about pre-optimizing these things since we have nothing in the pipeline that changes the payload schema, cause you would have to go in and remove it if we never support it. 😄